math/UsefulMathFunctions.cpp

```diff
@@ -146,43 +166,92 @@ void generatePrimes(int n) {
 // primeDivs[i] will contains all the prime divisors of number i.
 // O(n.log(log(n)))
 vector<int> primeDivs[N];
-void generatePrimeDivisors(int n) {
-    for (int i = 2; i <= n; ++i) {
-        if (primeDivs[i].size()) continue;
+void generatePrimeDivisors(int n)
+{
+    for (int i = 2; i <= n; ++i)
+    {
+        if (primeDivs[i].size())
+            continue;
 
-        for (int j = i; j <= n; j += i) {
+        for (int j = i; j <= n; j += i)
+        {
             primeDivs[j].push_back(i);
         }
     }
 }
 
 // Returns a list of divisors of the given number.
 // O(sqrt(n))
-vector<int> getDivisors(int n) {
-    vector<int> divs;
+vector<int> divsors;
+void getDivisors(int n)
+{
 
-    for (int i = 1; i * i <= n; ++i) {
-        if (n % i == 0) {
-            divs.push_back(i);
+    for (int i = 1; i * i <= n; ++i)
+    {
+        if (n % i == 0)
+        {
+            divsors.push_back(i);
 
-            if (i * i != n) {
-                divs.push_back(n / i);
+            if (i * i != n)
+            {
+                divsors.push_back(n / i);
             }
         }
     }
 
-    sort(divs.begin(), divs.end());
+    sort(divsors.begin(), divsors.end());
 
-    return divs;
+    return;
 }
 
 // Generates all the divisors of the numbers from 1 to n.
 // After calling this function,
 // divs[i] will contains all the divisors of number i.
 // O(n.log(n))
 vector<int> divs[N];
-void generateDivisors(int n) {
+void generateDivisors(int n)
+{
     for (int i = 1; i <= n; ++i)
         for (int j = i; j <= n; j += i)
             divs[j].push_back(i);
 }
+int main()
+{
+    cout << "GCD 15 35 :"
+         << " " << gcd(15, 35) << " " << gcd_rec(15, 35) << endl;
+    cout << "LCM 15 35 :"
+         << " " << lcm(15, 35) << endl;
+    cout << "ExtendedEuclid(15,35)" << extendedEuclid(15, 35).first << " " << extendedEuclid(15, 35).second << endl;
+    cout << "3^5 " << power(3, 5, MOD) << " " << power_rec(3, 5, MOD) << endl;
+    cout << "ModeInverse 92 " << modInverse(92, 7) << endl;
```

Comment:
Put in the output message the Modulo (7) 
```ModeInverse 92 7``` --> Output in console

Reply:
Done